code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -887,12 +887,11 @@
 /datum/reagent/lipolicide/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
 	if(!M.nutrition)
-		switch(rand(1,3))
-			if(1)
-				to_chat(M, "<span class='warning'>You feel hungry...</span>")
-			if(2)
-				update_flags |= M.adjustToxLoss(1, FALSE)
-				to_chat(M, "<span class='warning'>Your stomach grumbles painfully!</span>")
+		if(prob(33))
+			to_chat(M, "<span class='warning'>You feel hungry...</span>")
+		else if(prob(33))
```

Comment:
Might be my morning brain. But this won't lead to the exact odds as before.
```suggestion
		else if(prob(50))
```
50% chance on a 66% to get into this step will result in a 33% chance to get this result.
So each option here has a 33% chance of happening in the end.

Reply:
Hm, this doesn't make sense to me 😅 

My thought process here is that `rand(1, 3)` gives us a ~33% chance of getting each of the numbers 1, 2 and 3. With old code there is a ~66% chance of something happening, and ~33% of something not happening.

So I just took the 66% and split it up into two 33%'s.

Reply:
The no op looks a bit wack to me. Wackier than a switch with not all cases covered.
Unsure what maints think of it though.

For the probability stuff, you can try to imagine 100 coins. If you want exactly 33 of them (1/3 rounded down) you will take 1/3 of it (`prob(33)`).
Then the remainder is 67. Taking half of that will be 33 (rounded down) so `prob(50)` in our case.

I however do agree that this is less intuitive at first glance...

Reply:
use `pass()` if you need a no op